tardis/plasma/properties/nlte_excitation_data.py

```diff
@@ -0,0 +1,47 @@
+import logging
+
+import numpy as np
+
+logger = logging.getLogger(__name__)
+
+
+class NLTEExcitationData(object):
```

Comment:
Most of this class is code from the `NLTEData` class in `io/atom_data/base.py`. Is it possible to minimize the code duplication by sharing some of the methods?

Reply:
issue

Reply:
Described in #2266.